Matveibos/hometask-3/rainfall_stat.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# this class is used for calculate mean and variance values for town rainfall
+class RainfallStat
+  SEPARATOR = ':'
+  CHOOSING_NUMBERS = /\d*\.\d/.freeze
+
+  def initialize(town, data)
+    @town = town
+    @data = data
+    @processed_data = {}
```

Comment:
Maybe we can modify `create_hash` method (btw change its name to something like `process_data`) to return processed data that we will store in the `@processed_data` variable? Wdyt?